server/src/controller/ReviewsRouter.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const router = express.Router();
+const reviewsService = require('../service/ReviewsService');
+
+// CREATE
+// New Review Post
+router.post('/reviewPost', async (req, res) => {
+    //add username to body through the jwt tokens username
+    const data = await reviewsService.createNewReview(req.body);
```

Comment:
There should be error handling for when DynamoDB throws an error.  We can then catch any unforeseen errors/issues and give a status code 500.